jukeInTheBoxUti/www/src/File.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div>
+    <h1>Juke in the box</h1>
+    <div class="file">
+      <section>
+        <div class="file_piste">
+          <p v-cloak class="musique_file" v-for="(musique,index) in listMusiques">
+            {{index+1}})
+            <span v-for="(artiste,index) in musique.piste.artistes">
```

Comment:
J'ai un avertissement en console : "elements in iteration expect to have 'v-bind:key' directives".